packages/@o3r/create/src/index.ts

```diff
@@ -119,8 +120,26 @@ const addOtterCore = (relativeDirectory = '.') => {
   }
 };
 
+/**
+ * When the monorepo is created, the package json will keep only the ng script
+ * @param relativeDirectory package.json directory
+ */
+const cleanPackageJsonScripts = (relativeDirectory = '.') => {
```

Comment:
Should avoid as much as possible to do content change in this script.
I would suggest to do the change of the `package.json` (maybe by removing the specific scripts that does not make explicitly sense, not replacing the whole `script` field) in the `ng add @or3/core`

Reply:
I agree that it is not a good idea to complexify the `create`. After discussing the solution you proposed to remove the specific scripts it's what we need. I'll do the updates accordingly